src/components/WwwFrame/TheHeader.vue

```diff
@@ -559,6 +559,15 @@ export default {
 				&& this.$route.fullPath !== '/'
 				&& this.redirectToLoginExperimentVersion !== 'b';
 		},
+		hidePromoCreditBanner() {
```

Comment:
New mechanism to prevent visibility of this banner when not needed. There are likely other contexts we could add here.